README.md

```diff
@@ -516,6 +517,31 @@ iban.country_code             # => "AU"
 iban.branch_code              # => "123456"
 iban.account_number           # => "123456789"
 iban.iban                     # => nil
+
+# New Zealand
+iban = Ibandit::IBAN.new(
+  country_code: 'NZ',
+  bank_code: '01',
+  branch_code: '5659',
+  account_number: '3333333-44' # 7 digit account number and 2/3-digit account suffix
```

Comment:
We need the suffix separately for submission.

Reply:
had some thoughts around this too, but couldn't come up with an approach that didn't involve special-casing a huge portion of the existing library logic in order to expose an `account_number_suffix` field which would be `nil` in most cases (for now, at least, there may be other countries with this kind of behaviour). 

We already do something a little similar for Spanish bank accounts (with their 2-digit control numbers/check digits -- which is different from the 2-digit IBAN check digits), where rather than exposing a `account_control_number` field, the output of `spanish_iban.account_number` returns a 12-digit string with the first 2 being the `control digit` and the remaining 10 being the actual `account_number`. 

I understand this is not exactly the same thing, given that Spain has actual IBANs, that can be used in bank submission files, but if Ibandit guarantees that `nz_iban.account_number` always returns a 10-digit string, do you think it would be sufficient to do `nz_iban.account_number.last(3)` to extract the suffix when we need to?

Reply:
I would be happy adding a method on the IBAN that allows us to pull out the suffix without changing the code to allow it to be passed in separately. 

Eg.
```
nz_iban = Ibandit.new(country: 'NZ', bank_code: '01', branch_code: '1234', account_number: '1234555-22')

nz_iban.suffix  => '22'
```

This approach means we can change how we represent the number in just one place. I.e we decide we want to always have it one prefixes we just change it so `suffix => '022'` 

For IBAN's with other country codes we would return either `''`, `nil` or raise `NotImplementedError` (Maybe we add it for Spanish Iban's based on the comment above.)